src/puppetlabs/trapperkeeper/services/webserver/jetty9_core.clj

```diff
@@ -702,8 +704,9 @@
          enable-trailing-slash-redirect? (:enable-trailing-slash-redirect? options)
          normalize-request-uri? (:normalize-request-uri? options)]
      (.setBaseResource handler (Resource/newResource base-path))
-     (when follow-links?
-       (.addAliasCheck handler (AllowSymLinkAliasChecker.)))
+     (if follow-links?
+       (.addAliasCheck handler (AllowSymLinkAliasChecker.))
```

Comment:
The follow-links test passes, so jetty seems to handle more than one alias checker just fine, but I'm happy to change it to setAliasCheck instead.